src/transactions/apiTestHelpers.ts

```diff
@@ -0,0 +1,45 @@
+import type { EnhancedStore, Middleware, Reducer, UnknownAction } from '@reduxjs/toolkit'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+import { ApiReducersKeys } from 'src/redux/apiReducersList'
+import { RootState } from 'src/redux/reducers'
+import { RecursivePartial } from 'test/utils'
+
+// https://medium.com/@johnmcdowell0801/testing-rtk-query-with-jest-cdfa5aaf3dc1
+export function setupApiStore<
+  A extends {
+    reducer: Reducer<any, any>
+    reducerPath: string
+    middleware: Middleware
+    util: { resetApiState(): any }
+  },
+  Preloaded extends RecursivePartial<Omit<RootState, ApiReducersKeys>>,
+  R extends Record<string, Reducer<any, any>> = Record<never, never>,
+>(api: A, preloadedState: Preloaded, extraReducers?: R) {
+  const getStore = () =>
+    configureStore({
+      preloadedState,
+      reducer: combineReducers({
+        [api.reducerPath]: api.reducer,
+        ...extraReducers,
+      }),
+      middleware: (gdm) =>
+        gdm({ serializableCheck: false, immutableCheck: false }).concat(api.middleware),
```

Comment:
what's `gdm`?

Reply:
@jeanregisser Ah, that's `getDefaultMiddleware`. I've copied most of this function from this [helper function](https://github.com/reduxjs/redux-toolkit/blob/e7540a5594b0d880037f2ff41a83a32c629d3117/packages/toolkit/src/tests/utils/helpers.tsx#L186) and missed to rename this abbreviation. Will fix!